src/main/java/org/darisadesigns/polyglotlina/DictCore.java

```diff
@@ -261,18 +189,10 @@ public void pushUpdateWithCore(DictCore _core) {
         }
 
         // null root window indicates that this is a virtual dict core used for library analysis
-        if (rootWindow != null) {
-            rootWindow.updateAllValues(_core);
-        }
-    }
-
-    /**
-     * Returns root window of PolyGlot
-     *
-     * @return
-     */
-    public ScrMainMenu getRootWindow() {
-        return rootWindow;
+        // TODO: replace with signal?
```

Comment:
This is **critical** functionality. We'll need to figure out a solution to this before the code can be merged to master.